cross-chain/solana/scripts/deploy.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -eo pipefail
+
+# Setting env variables in the current bash shell
+source solana.env
+
+[ -z "$CLUSTER" ] && {
+  echo "'--cluster' option not provided" >&2
+  help
+  exit 1
+}
+
+[ -z "$AUTHORITY" ] && {
+  echo "'AUTHORITY' env var is not set" >&2
+  exit 1
+}
+
+echo "Building workspace for cluster: $CLUSTER ..."
+anchor build --provider.cluster $CLUSTER
+
+echo "Syncing the program's id ..."
+anchor keys sync
+
+echo "Building workspace again to include new program ID in the binary ..."
+anchor build --provider.cluster $CLUSTER
+
+echo "Deploying program(s) for cluster: $CLUSTER ..."
+anchor deploy --provider.cluster $CLUSTER --provider.wallet $AUTHORITY
+
+echo "Migrating..."
+anchor migrate --provider.cluster $CLUSTER --provider.wallet $AUTHORITY
```

Comment:
I've actually never used this, but this might be an interesting route to initialize your program. But because you have `deploy.ts` you don't need this step.

Whenever I've written deployment scripts, I've just written everything in a Typescript file, which I would execute via `ts-node` (like how you have your `deploy.ts` written up).

Reply:
Ok, I've created `/deploy` dir and put init scripts there. I'm calling the init scripts from the Makefile